<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="domingo">
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>285</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Domingo</string>
   </property>
  </widget>
  <widget class="QLabel" name="sabado">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sabado</string>
   </property>
  </widget>
  <widget class="QLabel" name="lunes">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>25</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lunes</string>
   </property>
  </widget>
  <widget class="QLabel" name="martes">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>155</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Martes</string>
   </property>
  </widget>
  <widget class="QLabel" name="jueves">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>415</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Jueves</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="boton_agghorario">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>580</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="viernes">
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>25</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Viernes</string>
   </property>
  </widget>
  <widget class="QLabel" name="miercoles">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>285</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Miercoles</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_lunes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_martes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_miercoles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_jueves">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_viernes">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_sabado">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="esp_domingo">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
